2022-09-27 22:24:54.558 Trying to detect encoding from a tiny portion of (16) byte(s).
2022-09-27 22:24:54.562 ascii passed initial chaos probing. Mean measured chaos is 0.000000 %
2022-09-27 22:24:54.563 ascii should target any language(s) of ['Latin Based']
2022-09-27 22:24:54.565 ascii is most likely the one. Stopping the process.

  You can now view your Streamlit app in your browser.

  Network URL: http://10.100.100.227:8501
  External URL: http://190.121.155.227:8501

/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
cp: cannot stat '/home/adagudelPDF_Images/nomap.png': No such file or directory
mv: cannot stat '/home/adagudelo/reporte_sismo/down/Images/nomap.png': No such file or directory
2022-09-27 22:25:03.364 Traceback (most recent call last):
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
  File "/home/jsoriano/Reposites/report_event_ang/GUI2.py", line 854, in <module>
    main()
  File "/home/jsoriano/Reposites/report_event_ang/GUI2.py", line 424, in main
    plt_inf.inf_ip()
  File "/home/jsoriano/Reposites/report_event_ang/plot_info.py", line 252, in inf_ip
    st.image(f"{folder}/Images/map_int_perc_{ID_event}.png")
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/elements/image.py", line 120, in image
    marshall_images(
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/elements/image.py", line 370, in marshall_images
    proto_img.url = image_to_url(
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/elements/image.py", line 286, in image_to_url
    with open(image, "rb") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/jsoriano/Reposites/report_event_ang/Events/SGC2022dugzug/Images/map_int_perc_SGC2022dugzug.png'

/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'xylon.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'xylon.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'xylon.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
[9009, 456, 456, 24, 7, 'Daño \nmoderado', 'Arjona', 'Astrea, Cesar', 'Floridablanca, Cúcuta, Cartagena De Indias, Medellín, Valledupar, Bucaramanga', 'Puerto Tejada, Guachené, Garzón', 'https://www.sgc.gov.co/detallesismo/SGC2020hlpcue/cdi']
True
aún no existe el json inf_general_.json
aún no existe el json inf_general_.json
aún no existe el json inf_general_.json
Normal
aún no existe el json inf_intpercibida_SGC2019zgudxj.json
aún no existe el json inf_intpercibida_SGC2019zgudxj.json
aún no existe el json inf_intpercibida_SGC2019zgudxj.json
aún no existe el json inf_aceleracion_SGC2019zgudxj.json
aún no existe el json inf_mecanismofocal_SGC2019zgudxj.json
aún no existe el json inf_mecanismofocal_SGC2019zgudxj.json
aún no existe el json inf_mecanismofocal_SGC2019zgudxj.json

aún no existe el json inf_aceleracion_SGC2019zgudxj.json
aún no existe el json inf_aceleracion_SGC2019zgudxj.json
aún no existe el json inf_instrumental_SGC2019zgudxj.json
aún no existe el json inf_instrumental_SGC2019zgudxj.json
aún no existe el json inf_intpercibida_SGC2019zgudxj.json
aún no existe el json inf_intpercibida_SGC2019zgudxj.json
[6985, 490, 490, 22, 6, 'Daño \nleve', 'Ataco', 'Ataco, Tolima', 'Pereira, Medellín, Ibagué, Villavicencio, Santiago De Cali, Bogotá, D.C.', 'Soledad, Barranquilla, Valle De Gaira', 'https://www.sgc.gov.co/detallesismo/SGC2019zgudxj/cdi']
<class 'reportlab.platypus.tables.Table'>
['__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', '_addCommand', '_argH', '_argW', '_bkgrndcmds', '_calc', '_calcNoSplitRanges', '_calcPreliminaryWidths', '_calcSpanRanges', '_calcSpanRects', '_calc_height', '_calc_width', '_canGetWidth', '_cellListIter', '_cellListProcess', '_cellStyles', '_cellvalues', '_colWidths', '_cr_0', '_cr_1_0', '_cr_1_1', '_culprit', '_doctemplate', '_doctemplateAttr', '_drawBkgrnd', '_drawBox', '_drawCell', '_drawGrid', '_drawHLines', '_drawHLinesB', '_drawInnerGrid', '_drawLines', '_drawOn', '_drawUnknown', '_drawVLines', '_drawVLinesA', '_elementWidth', '_fixedHeight', '_fixedWidth', '_frameName', '_getFirstPossibleSplitRowPosition', '_getRowImpossible', '_hAlignAdjust', '_hasVariWidthElements', '_linecmds', '_listCellGeom', '_listValueWidth', '_longTableOptimize', '_makeRoundedCornersClip', '_minRowHeights', '_ncols', '_nosplitCmds', '_nrows', '_prepLine', '_restoreRoundingObscuredLines', '_rowHeights', '_rowSplitRange', '_setCornerRadii', '_spanCmds', '_splitRows', '_srflcmds', 'draw', 'drawOn', 'getKeepWithNext', 'getSpaceAfter', 'getSpaceBefore', 'hAlign', 'ident', 'identity', 'isIndexing', 'minWidth', 'normCellRange', 'normalizeData', 'onSplit', 'repeatCols', 'repeatRows', 'setStyle', 'split', 'splitByRow', 'splitOn', 'vAlign', 'wrap', 'wrapOn']
aún no existe el json inf_general_.json
aún no existe el json inf_mecanismofocal_SGC2022syfalp.json
aún no existe el json inf_aceleracion_SGC2022syfalp.json
aún no existe el json inf_aceleracion_SGC2022syfalp.json
aún no existe el json inf_aceleracion_SGC2022syfalp.json
aún no existe el json inf_general_.json
[6985, 490, 490, 22, 6, 'Daño \nleve', 'Ataco', 'Ataco, Tolima', 'Pereira, Medellín, Ibagué, Villavicencio, Santiago De Cali, Bogotá, D.C.', 'Soledad, Barranquilla, Valle De Gaira', 'https://www.sgc.gov.co/detallesismo/SGC2019zgudxj/cdi']
<class 'reportlab.platypus.tables.Table'>
['__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', '_addCommand', '_argH', '_argW', '_bkgrndcmds', '_calc', '_calcNoSplitRanges', '_calcPreliminaryWidths', '_calcSpanRanges', '_calcSpanRects', '_calc_height', '_calc_width', '_canGetWidth', '_cellListIter', '_cellListProcess', '_cellStyles', '_cellvalues', '_colWidths', '_cr_0', '_cr_1_0', '_cr_1_1', '_culprit', '_doctemplate', '_doctemplateAttr', '_drawBkgrnd', '_drawBox', '_drawCell', '_drawGrid', '_drawHLines', '_drawHLinesB', '_drawInnerGrid', '_drawLines', '_drawOn', '_drawUnknown', '_drawVLines', '_drawVLinesA', '_elementWidth', '_fixedHeight', '_fixedWidth', '_frameName', '_getFirstPossibleSplitRowPosition', '_getRowImpossible', '_hAlignAdjust', '_hasVariWidthElements', '_linecmds', '_listCellGeom', '_listValueWidth', '_longTableOptimize', '_makeRoundedCornersClip', '_minRowHeights', '_ncols', '_nosplitCmds', '_nrows', '_prepLine', '_restoreRoundingObscuredLines', '_rowHeights', '_rowSplitRange', '_setCornerRadii', '_spanCmds', '_splitRows', '_srflcmds', 'draw', 'drawOn', 'getKeepWithNext', 'getSpaceAfter', 'getSpaceBefore', 'hAlign', 'ident', 'identity', 'isIndexing', 'minWidth', 'normCellRange', 'normalizeData', 'onSplit', 'repeatCols', 'repeatRows', 'setStyle', 'split', 'splitByRow', 'splitOn', 'vAlign', 'wrap', 'wrapOn']
aún no existe el json inf_general_.json
[6985, 490, 490, 22, 6, 'Daño \nleve', 'Ataco', 'Ataco, Tolima', 'Pereira, Medellín, Ibagué, Villavicencio, Santiago De Cali, Bogotá, D.C.', 'Soledad, Barranquilla, Valle De Gaira', 'https://www.sgc.gov.co/detallesismo/SGC2019zgudxj/cdi']
<class 'reportlab.platypus.tables.Table'>
['__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__weakref__', '_addCommand', '_argH', '_argW', '_bkgrndcmds', '_calc', '_calcNoSplitRanges', '_calcPreliminaryWidths', '_calcSpanRanges', '_calcSpanRects', '_calc_height', '_calc_width', '_canGetWidth', '_cellListIter', '_cellListProcess', '_cellStyles', '_cellvalues', '_colWidths', '_cr_0', '_cr_1_0', '_cr_1_1', '_culprit', '_doctemplate', '_doctemplateAttr', '_drawBkgrnd', '_drawBox', '_drawCell', '_drawGrid', '_drawHLines', '_drawHLinesB', '_drawInnerGrid', '_drawLines', '_drawOn', '_drawUnknown', '_drawVLines', '_drawVLinesA', '_elementWidth', '_fixedHeight', '_fixedWidth', '_frameName', '_getFirstPossibleSplitRowPosition', '_getRowImpossible', '_hAlignAdjust', '_hasVariWidthElements', '_linecmds', '_listCellGeom', '_listValueWidth', '_longTableOptimize', '_makeRoundedCornersClip', '_minRowHeights', '_ncols', '_nosplitCmds', '_nrows', '_prepLine', '_restoreRoundingObscuredLines', '_rowHeights', '_rowSplitRange', '_setCornerRadii', '_spanCmds', '_splitRows', '_srflcmds', 'draw', 'drawOn', 'getKeepWithNext', 'getSpaceAfter', 'getSpaceBefore', 'hAlign', 'ident', 'identity', 'isIndexing', 'minWidth', 'normCellRange', 'normalizeData', 'onSplit', 'repeatCols', 'repeatRows', 'setStyle', 'split', 'splitByRow', 'splitOn', 'vAlign', 'wrap', 'wrapOn']
aún no existe el json inf_general_.json
[6985, 490, 490, 22, 6, 'Daño \nleve', 'Ataco', 'Ataco, Tolima', 'Pereira, Medellín, Ibagué, Villavicencio, Santiago De Cali, Bogotá, D.C.', 'Soledad, Barranquilla, Valle De Gaira', 'https://www.sgc.gov.co/detallesismo/SGC2019zgudxj/cdi']
<class 'reportlab.platypus.tables.Table'>
2022-09-27 23:14:10.789 Traceback (most recent call last):
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
  File "/home/jsoriano/Reposites/report_event_ang/GUI2.py", line 854, in <module>
    main()
  File "/home/jsoriano/Reposites/report_event_ang/GUI2.py", line 424, in main
    plt_inf.inf_ip()
  File "/home/jsoriano/Reposites/report_event_ang/plot_info.py", line 252, in inf_ip
    st.image(f"{folder}/Images/map_int_perc_{ID_event}.png")
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/elements/image.py", line 120, in image
    marshall_images(
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/elements/image.py", line 370, in marshall_images
    proto_img.url = image_to_url(
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/elements/image.py", line 286, in image_to_url
    with open(image, "rb") as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/jsoriano/Reposites/report_event_ang/Events/SGC2021dnegqn/Images/map_int_perc_SGC2021dnegqn.png'

/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'xylon.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'xylon.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'xylon.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
cp: cannot stat '/home/adagudelPDF_Images/nomap.png': No such file or directory
mv: cannot stat '/home/adagudelo/reporte_sismo/down/Images/nomap.png': No such file or directory
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
cp: cannot stat '/home/adagudelPDF_Images/nomap.png': No such file or directory
mv: cannot stat '/home/adagudelo/reporte_sismo/down/Images/nomap.png': No such file or directory
2022-12-03 13:47:52.713 Trying to detect encoding from a tiny portion of (16) byte(s).
2022-12-03 13:47:52.753 ascii passed initial chaos probing. Mean measured chaos is 0.000000 %
2022-12-03 13:47:52.754 ascii should target any language(s) of ['Latin Based']
2022-12-03 13:47:52.754 ascii is most likely the one. Stopping the process.

  You can now view your Streamlit app in your browser.

  Network URL: http://10.100.100.227:8501
  External URL: http://190.121.155.227:8501

mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
/home/jsoriano/Reposites/report_event_sor/web_inf.py:522: UserWarning: Tight layout not applied. The bottom and top margins cannot be made large enough to accommodate all axes decorations. 
  fig.tight_layout()
2022-12-03 15:39:35.754 Trying to detect encoding from a tiny portion of (16) byte(s).
2022-12-03 15:39:35.756 ascii passed initial chaos probing. Mean measured chaos is 0.000000 %
2022-12-03 15:39:35.757 ascii should target any language(s) of ['Latin Based']
2022-12-03 15:39:35.757 ascii is most likely the one. Stopping the process.

  You can now view your Streamlit app in your browser.

  Network URL: http://10.100.100.227:8501
  External URL: http://190.121.155.227:8501

mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
False
aún no existe el json inf_general_.json
aún no existe el json inf_general_.json
aún no existe el json inf_aceleracion_SGC2022xtrggr.json
aún no existe el json inf_general_SGC2022.json
aún no existe el json inf_general_SGC2022xtr.json
aún no existe el json inf_aceleracion_SGC2022xtrggr.json
aún no existe el json inf_aceleracion_SGC2022xtrggr.json
aún no existe el json inf_general_.json
aún no existe el json inf_general_.json
aún no existe el json inf_general_.json
[110, 12, 12, 4, 5, 'Sentido \nfuertemente', 'Cúcuta', 'Cúcuta, Norte De Santander', 'Bucaramanga, Bucaramanga, El Zulia, El Zulia, Los Patios, Cúcuta', 'Piedecuesta, Medellín, Bogotá, D.C.', 'https://www.sgc.gov.co/detallesismo/SGC2022xssict/cdi']
aún no existe el json inf_general_.json
aún no existe el json inf_general_.json
aún no existe el json inf_general_.json
aún no existe el json inf_general_.json
aún no existe el json inf_general_.json
  Stopping...

  Git integration is disabled.

  Streamlit requires Git 2.7.0 or later, but you have 1.9.1.
  Git is used by Streamlit Cloud (https://streamlit.io/cloud).
  To enable this feature, please update Git.
2023-01-29 22:41:43.701 Trying to detect encoding from a tiny portion of (16) byte(s).
2023-01-29 22:41:43.767 ascii passed initial chaos probing. Mean measured chaos is 0.000000 %
2023-01-29 22:41:43.768 ascii should target any language(s) of ['Latin Based']
2023-01-29 22:41:43.768 ascii is most likely the one. Stopping the process.

  You can now view your Streamlit app in your browser.

  Network URL: http://10.100.100.227:8501
  External URL: http://190.121.155.227:8501

mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
/home/jsoriano/Reposites/report_event_sor/web_inf.py:522: UserWarning: Tight layout not applied. The bottom and top margins cannot be made large enough to accommodate all axes decorations. 
  fig.tight_layout()
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
/home/jsoriano/Reposites/report_event_sor/web_inf.py:522: UserWarning: Tight layout not applied. The bottom and top margins cannot be made large enough to accommodate all axes decorations. 
  fig.tight_layout()
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
2023-02-01 00:47:55.027 Traceback (most recent call last):
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
  File "/home/jsoriano/Reposites/report_event_sor/GUI2.py", line 6, in <module>
    import mkPDF as mkPDF
  File "/home/jsoriano/Reposites/report_event_sor/mkPDF.py", line 553
    table_pi = Table(Table_Data_pi,colWidths=[4.2*cm, 3.6*cm],
    ^
SyntaxError: invalid syntax

mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
2023-02-01 00:50:26.222 Traceback (most recent call last):
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
  File "/home/jsoriano/Reposites/report_event_sor/GUI2.py", line 910, in <module>
    main()
  File "/home/jsoriano/Reposites/report_event_sor/GUI2.py", line 886, in main
    mkPDF.mkPDF_report(ID_event)
  File "/home/jsoriano/Reposites/report_event_sor/mkPDF.py", line 553, in mkPDF_report
    table_pi = Table(Table_Data_pi,colWidths=[4.2*cm, 3.6*cm],
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/reportlab/platypus/tables.py", line 308, in __init__
    raise ValueError("%s data error - %d rows in data but %d in row heights" % (self.identity(),nrows, len(rowHeights)))
ValueError: <Table@0x7FD000B50CA0 7 rows x 8 cols> with cell(0,0) containing
'Número de reportes recibidos' data error - 7 rows in data but 13 in row heights

2023-02-01 00:50:32.274 Traceback (most recent call last):
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
  File "/home/jsoriano/Reposites/report_event_sor/GUI2.py", line 910, in <module>
    main()
  File "/home/jsoriano/Reposites/report_event_sor/GUI2.py", line 886, in main
    mkPDF.mkPDF_report(ID_event)
  File "/home/jsoriano/Reposites/report_event_sor/mkPDF.py", line 553, in mkPDF_report
    table_pi = Table(Table_Data_pi,colWidths=[4.2*cm, 3.6*cm],
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/reportlab/platypus/tables.py", line 308, in __init__
    raise ValueError("%s data error - %d rows in data but %d in row heights" % (self.identity(),nrows, len(rowHeights)))
ValueError: <Table@0x7FD002B80C10 7 rows x 8 cols> with cell(0,0) containing
'Número de reportes recibidos' data error - 7 rows in data but 13 in row heights

2023-02-01 00:52:24.150 Traceback (most recent call last):
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
  File "/home/jsoriano/Reposites/report_event_sor/GUI2.py", line 6, in <module>
    import mkPDF as mkPDF
  File "/home/jsoriano/Reposites/report_event_sor/mkPDF.py", line 545
    ["Descripción intensidad \nmáxima",EMS_98{str(int_maxima)}],
                                               ^
SyntaxError: invalid syntax

mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
2023-02-01 00:54:59.404 Traceback (most recent call last):
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
  File "/home/jsoriano/Reposites/report_event_sor/GUI2.py", line 910, in <module>
    main()
  File "/home/jsoriano/Reposites/report_event_sor/GUI2.py", line 886, in main
    mkPDF.mkPDF_report(ID_event)
  File "/home/jsoriano/Reposites/report_event_sor/mkPDF.py", line 545, in mkPDF_report
    ["Descripción intensidad \nmáxima",EMS_98[str(int_maxima)]],
KeyError: '4'

mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
/home/jsoriano/Reposites/report_event_sor/web_inf.py:522: UserWarning: Tight layout not applied. The bottom and top margins cannot be made large enough to accommodate all axes decorations. 
  fig.tight_layout()
2023-02-03 23:22:01.600 Traceback (most recent call last):
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
  File "/home/jsoriano/Reposites/report_event_sor/GUI2.py", line 909, in <module>
    main()
  File "/home/jsoriano/Reposites/report_event_sor/GUI2.py", line 885, in main
    mkPDF.mkPDF_report(ID_event)
  File "/home/jsoriano/Reposites/report_event_sor/mkPDF.py", line 114, in mkPDF_report
    with open(folder+"/Data/"+f"inf_general_{ID_event}.json","r") as json_file:
FileNotFoundError: [Errno 2] No such file or directory: 'Events/SGC2022ytfrww/Data/inf_general_SGC2022ytfrww.json'

/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'archive.sgc.gov.co'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/openpyxl/styles/stylesheet.py:226: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
/home/jsoriano/Reposites/report_event_sor/web_inf.py:522: UserWarning: Tight layout not applied. The bottom and top margins cannot be made large enough to accommodate all axes decorations. 
  fig.tight_layout()
mv: cannot move '/home/jsoriano/tableDownload.csv' to '/home/jsoriano/Tables/aceleracion_SGC2022tmqchk.csv': No such file or directory
2023-02-04 00:25:51.707 Traceback (most recent call last):
  File "/home/jsoriano/anaconda3/envs/report-env/lib/python3.9/site-packages/streamlit/script_runner.py", line 354, in _run_script
    exec(code, module.__dict__)
  File "/home/jsoriano/Reposites/report_event_sor/GUI2.py", line 12, in <module>
    import plot_info ###
  File "/home/jsoriano/Reposites/report_event_sor/plot_info.py", line 137
    print("nombre_estacion_max: " nombre_estacion_max)
                                  ^
SyntaxError: invalid syntax

